packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -290,6 +291,9 @@ public void receiveCommand(
       case FOCUS_TEXT_INPUT:
         this.receiveCommand(reactEditText, "focus", args);
         break;
+      case REQUEST_TV_FOCUS:
+        this.receiveCommand(reactEditText, "focus", args);
+        break;
```

Comment:
Mostly for consistency across views. So that we don't need to check if it's TextInput. 

Adding it on JS side won't support dispatchViewManagerCommand (that's what we are using internally to request focus). 

I think sticking to requestTVFocus across every component would be more predictable